ui/main.qml

```diff
@@ -443,39 +433,50 @@ StatusWindow {
         id: onboardingV2
 
         Onboarding2.OnboardingLayout {
+            id: onboardingLayout
             objectName: "startupOnboardingLayout"
             anchors.fill: parent
 
             // TODO implement those two
-            loginAccountsModel: ListModel {}
+            loginAccountsModel: onboardingStore.loginAccountsModel
             isBiometricsLogin: false
             networkChecksEnabled: true
             biometricsAvailable: Qt.platform.os === Constants.mac
 
-            onboardingStore: onboardingStoreLoader.item
+            onboardingStore: onboardingStore
 
             onFinished: (flow, data) => {
-                console.warn("!!! ONBOARDING FINISHED; flow:", flow, "; data:", JSON.stringify(data))
-
-                let error = onboardingStoreLoader.item.finishOnboardingFlow(flow, data)
+                let error = onboardingStore.finishOnboardingFlow(flow, data)
 
                 if (error != "") {
+                    // We should never be here since everything should be validated already
                     console.error("!!! ONBOARDING FINISHED WITH ERROR:", error)
-                    // TODO show error
                     return
                 }
```

Comment:
Yes, we don't need this check because of two resones
- everything should be validated
- even is not, nim side can reject the login as for any other login errors by emitting `accountLoginError`